atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172082 configure vitest to run tests for components

### What does this PR do?
configures the vitest chore for testing components
### Description of Task to be completed?
This PR allows the configuration of vitest to run tests of components
### How should this be manually tested?
After cloning the Repo user should create new storybook features and run them using `yarn run test` and for coverage use `yarn run coverage`
### What are the relevant pivotal tracker/Trello stories?
#185172082
